style.css

```diff
@@ -477,10 +477,10 @@ label {
 /* Modal Content */
 .modal-content {
   position: relative;
-  background: #a2a3ae;
+  background: #36347a;
   margin: auto;
   padding: 0;
-  border: 1px solid #888;
+  border: 1px solid #36347a;
   width: 80%;
   box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
   -webkit-animation-name: animatetop;
```

Comment:
## Your Modal Window
### Desktop

![popup-fixable](https://user-images.githubusercontent.com/93039359/181769911-bd2ca0f5-adbe-4df3-9c21-d16559fcdee0.png)

### Mobile
![mobile-popup-fixable](https://user-images.githubusercontent.com/93039359/181769877-04ad1f8b-eae3-403c-9fde-4f15f3c483aa.png)
## Figma Template
### Desktop
![popup-figma](https://user-images.githubusercontent.com/93039359/181770163-7e369833-cfb5-44b2-acc6-025503ba7894.png)

### Mobile
![popup2-figma](https://user-images.githubusercontent.com/93039359/181770199-45970ee2-b777-47f8-90a6-b21544e8098a.png)

- [ ] Your design of the modal is not following the Figma template. Please fix the background issue by giving it a width of 100% :blush: